pages/home.page.php

```diff
@@ -48,38 +48,56 @@
 		<h2 class="text-center sub-title"><?= $locales->RECENT_SPAWNS ?></h2>
-		
 		<div class="last-mon-js">
-		
-		<?php foreach ($recents as $pokemon) { ?>
-		
-			<div class="col-md-1 col-xs-4 pokemon-single" data-pokeid="<?= $pokemon ?>">
-			
-				<a href="pokemon/<?= $pokemon ?>"><img src="core/pokemons/<?= $pokemon ?>.png" alt="<?= $pokemons->pokemon->$pokemon->name ?>" class="img-responsive"></a>
-				<p class="pkmn-name"><a href="pokemon/<?= $pokemon ?>"><?= $pokemons->pokemon->$pokemon->name ?></a></p>
-			
+		<?php
+		foreach ($recents as $key => $pokemon) {
+			$id = $pokemon->id; ?>
+			<div class="col-md-1 col-xs-3 pokemon-single" data-pokeid="<?= $id ?>">
+				<a href="pokemon/<?= $id ?>"><img src="core/pokemons/<?= $id ?>.png" alt="<?= $pokemons->pokemon->$id->name ?>" class="img-responsive"></a>
+				<a href="pokemon/<?= $id ?>"><p class="pkmn-name"><?= $pokemons->pokemon->$id->name ?></p></a>
+				<a href="https://maps.google.com/?q=<?= $pokemon->last_location->latitude ?>,<?= $pokemon->last_location->longitude ?>&ll=<?= $pokemon->last_location->latitude ?>,<?= $pokemon->last_location->longitude ?>&z=16" target="_blank">
+					<small id="timer-<?= $key ?>-js">00:00:00</small>
+				</a>
+				<?php
+				if ($config->system->recents_show_iv) {
+					if ($pokemon->iv->available) { ?>
+					<div class="progress" style="height: 6px; width: 80%; margin: 5px auto 15px auto;">
+						<div title="Stamina IV: <?= $pokemon->iv->stamina ?>" class="progress-bar progress-bar-success" role="progressbar" aria-valuenow="<?= $pokemon->iv->stamina ?>" aria-valuemin="0" aria-valuemax="45" style="width: <?= ((100/15)*$pokemon->iv->stamina)/3 ?>%">
+							<span class="sr-only">Stamina IV: <?= $pokemon->iv->stamina ?></span>
+						</div>
+						<div title="Attack IV: <?= $pokemon->iv->attack ?>" class="progress-bar progress-bar-danger" role="progressbar" aria-valuenow="<?= $pokemon->iv->attack ?>" aria-valuemin="0" aria-valuemax="45" style="width: <?= ((100/15)*$pokemon->iv->attack)/3 ?>%">
+							<span class="sr-only">Attack IV: <?= $pokemon->iv->attack ?></span>
+						</div>
+						<div title="Defense IV: <?= $pokemon->iv->defense ?>" class="progress-bar progress-bar-info" role="progressbar" aria-valuenow="<?= $pokemon->iv->defense ?>" aria-valuemin="0" aria-valuemax="45" style="width: <?= ((100/15)*$pokemon->iv->defense)/3 ?>%">
+							<span class="sr-only">Defense IV: <?= $pokemon->iv->defense ?></span>
+						</div>
+					</div>
+					<?php
+					} else { ?>
+					<div class="progress" style="height: 6px; width: 80%; margin: 5px auto 15px auto;">
+						<div title="IV not available" class="progress-bar" role="progressbar" style="width: 100%; background-color:#CCCCCC;" aria-valuenow="1" aria-valuemin="0" aria-valuemax="1"></div>
+					</div>
+				<?php
+					}
+				} ?>
 			</div>
-			
-		<?php }?>
-		
+		<?php
+			// Array with ids and countdowns to start at the end of this file
+			$timers["#timer-".$key."-js"] = $pokemon->last_seen - time();
```

Comment:
I don't know whether this is a good idea to do it like this? (see below as well)